### PR TITLE
[Merged by Bors] - sync: update sha from backports

### PR DESCRIPTION
**Pull request by joneugster**
These files have been primarily modified by backports and need little modification:

* `topology.basic`: #1826 - modified with a porting note, which can now be removed
* `data.real.cau_seq_completion`: #1469 - not a backport, but forgot to update the SHA
* `order.filter.n_ary.basic`: #1967 - this PR forgot to update the SHA
* `ring_theory.valuation.basic`: The change is a small golf that is now included in this PR
---

* [`topology.basic`@`bcfa726826abd57587355b4b5b7e78ad6527b7e4`..`88b8a77d63a702923d9bee05e9e454ebc22aa766`](https://leanprover-community.github.io/mathlib-port-status/file/topology/basic?range=bcfa726826abd57587355b4b5b7e78ad6527b7e4..88b8a77d63a702923d9bee05e9e454ebc22aa766)
* [`data.real.cau_seq_completion`@`40acfb6aa7516ffe6f91136691df012a64683390`..`cf4c49c445991489058260d75dae0ff2b1abca28`](https://leanprover-community.github.io/mathlib-port-status/file/data/real/cau_seq_completion?range=40acfb6aa7516ffe6f91136691df012a64683390..cf4c49c445991489058260d75dae0ff2b1abca28)
* [`order.filter.n_ary`@`1f0096e6caa61e9c849ec2adbd227e960e9dff58`..`78f647f8517f021d839a7553d5dc97e79b508dea`](https://leanprover-community.github.io/mathlib-port-status/file/order/filter/n_ary?range=1f0096e6caa61e9c849ec2adbd227e960e9dff58..78f647f8517f021d839a7553d5dc97e79b508dea)
* [`ring_theory.valuation.basic`@`da420a8c6dd5bdfb85c4ced85c34388f633bc6ff`..`2196ab363eb097c008d4497125e0dde23fb36db2`](https://leanprover-community.github.io/mathlib-port-status/file/ring_theory/valuation/basic?range=da420a8c6dd5bdfb85c4ced85c34388f633bc6ff..2196ab363eb097c008d4497125e0dde23fb36db2)

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Can you locate the mathlib4 PRs that made these changes and link them in the description?

File: Mathlib/RingTheory/Valuation/Basic.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kevin Buzzard, Johan Commelin, Patrick Massot
 
 ! This file was ported from Lean 3 source module ring_theory.valuation.basic
-! leanprover-community/mathlib commit da420a8c6dd5bdfb85c4ced85c34388f633bc6ff
```
Comment (eric-wieser) [inline Mathlib/RingTheory/Valuation/Basic.lean:7]: I think there's a golf left to forward-port here

Comment (joneugster) [reply Mathlib/RingTheory/Valuation/Basic.lean:7]: my bad, added that now.

**Comment by eric-wieser** [comment]
bors merge

Thanks!

**Pull request closed without merging**